File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int>mp;
        for(int n=0; n<nums.size();n++){
            int ans=target-nums[n];
            if(mp.find(ans)!=mp.end()){
                return {mp[ans],n};
            }
            mp[nums[n]]=n;

        }
        return {};
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen={}
        ans=[]
        for i,num in enumerate(nums):
            comp=target-num
            if comp in seen:
                return[i,seen[comp]]
            else:
                seen[num]=i




        